workorders.lic

```diff
@@ -489,6 +489,55 @@ class WorkOrders
     end
   end
 
+  def gather_process_herb(herb, herb_to_purchase)
```

Comment:
What do you think about breaking this into a forage.lic script? I could find that useful outside of the workorders as well.

`;forage <quantity> <herb name> <rnum>`

Reply:
See updated comment.